lightningd/watch.c

```diff
@@ -270,19 +270,17 @@ void watch_topology_changed(struct chain_topology *topo)
 	struct txwatch_hash_iter i;
 	struct txwatch *w;
 	bool needs_rerun;
-
-again:
-	/* Iterating a htable during deletes is safe, but might skip entries. */
-	needs_rerun = false;
-	for (w = txwatch_hash_first(&topo->txwatches, &i);
-	     w;
-	     w = txwatch_hash_next(&topo->txwatches, &i)) {
-		u32 depth;
-
-		depth = get_tx_depth(topo, &w->txid);
-		if (depth)
-			needs_rerun |= txw_fire(w, &w->txid, depth);
-	}
-	if (needs_rerun)
-		goto again;
+	do {
+		/* Iterating a htable during deletes is safe, but might skip entries. */
+		needs_rerun = false;
+		for (w = txwatch_hash_first(&topo->txwatches, &i);
+	    	w;
+	    	w = txwatch_hash_next(&topo->txwatches, &i)) {
```

Comment:
nit: Above two lines should be tabbed over.  They are part of the for loop.

Reply:
Looks like they mix tabs and spaces

Reply:
I can quickly fix this again but in my editor it looked pretty good

Reply:
I know, but it doesn't in other editors that display tabs as 2, 4 or 8 spaces. Please configure your editor to use tabs instead :-)